packages/react/src/components/Table/TableSkeletonWithHeaders/TableSkeletonWithHeaders.jsx

```diff
@@ -79,7 +82,7 @@ const TableSkeletonWithHeaders = ({
           className={`${iotPrefix}--table-skeleton-with-headers--table-row`}
         >
           {hasRowSelection === 'multi' ? <TableCell /> : null}
-          {hasRowExpansion ? <TableCell /> : null}
+          {hasRowExpansion || hasRowNesting ? <TableCell /> : null}
           {loadingColumns.map((v, colIndex) => (
             <TableCell key={`emptycell-${colIndex}`}>
               {rowIndex === 0 && !columns.length ? <SkeletonText /> : null}
```

Comment:
Might this be a good time to use Shady's SkeletonRow from a couple week ago?

Reply:
I hear you and I started doing that, but realised it would be a slightly bigger refactoring (SkeletonRow is not based on TableRow and there are other small differences with classes etc) than the tiny and safe fix I promised David in order to add this one to the current sprint  :-)

Reply:
Ah, got it. No worries! Just wanted to check.

Reply:
We can create another issue to use SkeletonRow here so that it is not forgotten

Reply:
https://github.com/carbon-design-system/carbon-addons-iot-react/issues/3336